admin/package.json

```diff
@@ -8,14 +8,16 @@
     "start": "node ./server.js",
     "postinstall": "gulp build",
     "test": "jasmine-node spec --verbose --color --captureExceptions --forceexit",
-    "coverage": "nyc jasmine-node spec --verbose --color --captureExceptions --forceexit"
+    "coverage": "nyc jasmine-node spec --verbose --color --captureExceptions --forceexit",
+    "startn": "nodemon ./server.js",
+    "watch-jscss": "gulp watch"
```

Comment:
Good 👍 

But why is this called `watch-jscss`? What does the `j` mean?

Reply:
Most likely a typo

Reply:
Do you want to remove the `j` then?  Personally I would go with `sasswatch` as this is the name in the pupil-spa package.json